packages/dashboard/src/hooks/useAssetModel/describeAssetModelRequest.ts

```diff
@@ -0,0 +1,47 @@
+import { DescribeAssetModelCommand, type IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
+
+export class DescribeAssetModelRequest {
```

Comment:
this seems to be in the wrong folder ? this probably doesnt belong in hooks ?

Reply:
I'm using this hook in the resource explorer and the toolbar to list the assets for a model.